src/psyclone/tests/lfric_build.py

```diff
@@ -63,6 +63,7 @@ class LFRicBuild(Compile):
                               "argument_mod",
                               "kernel_mod",
                               "partition_mod",
+                              "reference_element_mod",
```

Comment:
Please update the years in the Copyright to "2017-2020".

Reply:
Done.